source/ncconvert/builder/Target.h

```diff
@@ -8,8 +8,17 @@ namespace nc::convert
 /** @brief Data describing a required asset conversion. */
 struct Target
 {
+    Target(std::filesystem::path source,
```

Comment:
Haven't figured out exactly why yet, but locally msvc is complaining about 'uninitialized memory' when this optional has a default initializer. Pipeline msvc version seems ok with it though... Adding a constructor makes both happy.